lib/hmmlearn/hmm.py

```diff
@@ -307,18 +309,246 @@ def _do_mstep(self, stats):
                                      (cvweight + stats['post'][:, None, None]))
 
 
-_MULTINOMIALHMM_DOC_SUFFIX = """
+
+class MultinomialHMM(BaseHMM):
+    """
+    Hidden Markov Model with multinomial emissions, e.g. BoW per state
```

Comment:
what is "BoW"?

Reply:
BoW is "bag of words" but I removed it since other HMM description don't go into these specifics.